integration/build.gradle

```diff
@@ -1,4 +1,5 @@
 repositories {
+    jcenter()
```

Comment:
JBoss repository seemed to experience some issues - see [this CI run](https://github.com/allegro/hermes/runs/3793959021?check_suite_focus=true). CI passed after adding jcenter as an alternative.